src/grid.py

```diff
@@ -16,103 +16,72 @@ class ItemNotFoundException(Exception):
         self.__cells[loc.y][loc.x] = None
         return content
 
-    def location_in_bounds(self, loc: Location):
+    def location_in_bounds(self, loc: Location) -> bool:
         return 0 <= loc.x <= self.width and 0 <= loc.y <= self.height
 
-    def move(self, source: Location, dest: Location):
-        item = self.clear(source)
-        self.place(item, dest)
+    def move(self, source: Location, dest: Location) -> None:
+        if item := self.clear(source):
+            self.place(item, dest)
 
-    def find(self, item) -> Location:
+    def find(self, item: GridItem) -> Location:
         try:
-            return self.__items_index.get(item.uuid)
+            return self.__items_index[item.uuid]
         except IndexError:
             raise ItemNotFoundException("Item not in grid")
 
     def get_adjacent_items(self, loc: Location) -> Set[GridItem]:
         all_adjacent_indices = set()
-        for x in range(loc.x-1, loc.x+2):
-            for y in range(loc.y-1, loc.y+2):
+        for x in range(loc.x - 1, loc.x + 2):
+            for y in range(loc.y - 1, loc.y + 2):
                 adj = Location(x, y)
                 if self.location_in_bounds(adj) and adj != loc:
-                    all_adjacent_indices.add(adj)
-
-        if loc.x == 0:
-            indices_left = {
-                Location(loc.x - 1, loc.y),
-                Location(loc.x - 1, loc.y + 1),
-                Location(loc.x - 1, loc.y - 1)
-            }
-            all_adjacent_indices = all_adjacent_indices - indices_left
-
-        if loc.x == len(self.__cells)-1:
-            indices_right = {
-                Location(loc.x + 1, loc.y),
-                Location(loc.x + 1, loc.y + 1),
-                Location(loc.x + 1, loc.y - 1)
-            }
-            all_adjacent_indices = all_adjacent_indices - indices_right
-
-        if loc.y == 0:
-            indices_above = {
-                Location(loc.x - 1, loc.y - 1),
-                Location(loc.x + 1, loc.y - 1),
-                Location(loc.x, loc.y - 1),
-            }
-            all_adjacent_indices = all_adjacent_indices - indices_above
-
-        if loc.y == len(self.__cells[0])-1:
-            indices_below = {
-                Location(loc.x - 1, loc.y + 1),
-                Location(loc.x + 1, loc.y + 1),
-                Location(loc.x, loc.y + 1),
-            }
-            all_adjacent_indices = all_adjacent_indices - indices_below
-
-        return {
-            self.__cells[adjacent_loc.y][adjacent_loc.x] for adjacent_loc in all_adjacent_indices
-            if self.__cells[adjacent_loc.y][adjacent_loc.x] is not None
-        }
-
-    def __repr__(self):
-        return tabulate(self.__cells, tablefmt='grid')
+                    if grid_item := self.__cells[x][y]:
+                        all_adjacent_indices.add(grid_item)
+        return all_adjacent_indices
+
+    def __repr__(self) -> str:
+        return tabulate(self.__cells, tablefmt="grid")
```

Comment:
Any specific reason you replaced to double quotes? Just curious

Reply:
That's what linting does. Probably this follows a recent PEP